File: file.js
const { twoSum } = require('./Solution.js');

function output() {
    const num1 = process.argv[2];
    const num2 = process.argv[3];

    const sum = twoSum(parseInt(num1), parseInt(num2));
    const fs = require('fs');
    const data = sum.toString();
    fs.writeFile('output.txt', data, (err) => {
        if (err) throw err;
    });
}

output();

File: file.java
import java.io.*;
// import Solution.*;
public class file {
    public static void main(String[] args) {
        int a, b;
        a = Integer.parseInt(args[0]);
        b = Integer.parseInt(args[1]);
        int sum = Solution.twoSum(a, b);
        try {
            FileWriter writer = new FileWriter("output.txt");
            writer.write("" + sum);
            writer.close();
        } catch (IOException e) {
            System.out.println("An error occurred.");
            e.printStackTrace();
        }
    }
}
